### PR TITLE
80 fix/scrutinizer js errors

### LINKED ISSUE #80: Fix JS issues
see https://scrutinizer-ci.com/g/sudar/bulk-move/issues/dev/2.0.0/files/assets/js/bulk-move.js?orderField=path&order=asc&honorSelectedPaths=0&issueId=27599311#inspectioncomment-98297358

see https://scrutinizer-ci.com/g/sudar/bulk-move/issues/dev/2.0.0/files/assets/js/bulk-move.js?orderField=path&order=asc&honorSelectedPaths=0&issueId=27774916#inspectioncomment-98297357

### PR DESCRIPTION
**Pull request by mariadanieldeepak**
Fix #80

### CODE REVIEW

File: assets/js/src/bulk-move.js [hunk 1]
```diff
@@ -55,14 +55,16 @@ jQuery(document).ready(function () {
             if (!valid) {
                 e.preventDefault();
                 errorKey = validator.replace('validate_', '');
-                alert(BULK_MOVE.error.errorKey);
+                alert(BULK_MOVE.error[ errorKey ]);
                 return false;
             }
+            return;
```
Comment (sudar) [inline assets/js/src/bulk-move.js]: We don't need to add empty `return` statements.

File: include/Page/Base.php [hunk 2]
```diff
@@ -229,7 +229,7 @@ public function modify_admin_footer_text( $footer_text ) {
 	 * Enqueue JavaScript.
 	 */
 	public function enqueue_script() {
-		wp_enqueue_script( 'bulk-move', $this->get_plugin_dir_url() . 'assets/js/bulk-move.js', array( 'jquery', 'postbox' ), BM_BulkMove::VERSION, true );
+		wp_enqueue_script( 'bulk-move', $this->get_plugin_dir_url() . 'assets/js/src/bulk-move.js', array( 'jquery', 'postbox' ), BM_BulkMove::VERSION, true );
```
Comment (sudar) [inline include/Page/Base.php:232]: We should include the js file from the `assets/js/` folder not from `src` folder.

We will have a build step (similar to Bulk Delete) where we will contact and compress files from `assets/js/src` folder into `assets/js` folder.

**Review by sudar — CHANGES_REQUESTED** [review]

File: assets/js/src/bulk-move.js (hunk 1 above)
Comment (mariadanieldeepak) [reply assets/js/src/bulk-move.js]: @sudar,

 > We don't need to add empty return statements.

I added it after reading Scrutinizer's comments. I'll remove it as it doesn't add value.

File: include/Page/Base.php (hunk 2 above)
Comment (mariadanieldeepak) [reply include/Page/Base.php:232]: @sudar,

> We will have a build step (similar to Bulk Delete) where we will contact and compress files from assets/js/src folder into assets/js folder.

In this case, can we reference the minified file than the actual JS file? We minify it anyway. (I've referenced the bulk-move.min.js file)

**Comment by mariadanieldeepak** [comment]
@sudar,

Please note that the Scrutinizer would fail again asking for the explicit `return` statements.

**Review by sudar — APPROVED** [review]

**Pull request merged**